scripts/check-and-run-apollo-cli.sh

```diff
@@ -1,3 +1,5 @@
+echo "warning: The check-and-run-apollo-cli.sh script is deprecated and will be removed. Please see https://www.apollographql.com/docs/ios/installation/#adding-a-code-generation-build-step for updated build instructions which use a bundled version of the Apollo CLI."
```

Comment:
That's why I prefaced this with `warning`, so it would show up in the Xcode sidebar and not just in the build log like so: 

<img width="334" alt="Screen Shot 2019-09-20 at 4 30 13 PM" src="https://user-images.githubusercontent.com/1976498/65360101-3c1b3600-dbc4-11e9-87ec-b6873fce8006.png">

Reply:
Ah, cool, didn't know about that :+1: